nueljoe/healthcare
# feature(course) course enrollment

#### What does this PR do?
Implement course enrollment and payment endpoints

#### Description of Task to be completed?
- implement the `POST /courses/:slug/enrollments` endpoint to initiate the enrollment of the current user in a course
- integrate with a payment gateway
- implement the `POST /payments/verification` endpoint to allow the server receive feedback from the integrated payment gateway on the status of a payment
- implement the `GET /payments` endpoint to fetch the entire payment history on the platform
- add a `price` column and a `discount` column to the courses table in order to allow courses to be priced and discounted individually.

#### How should this be manually tested?
- clone the repository
- cd into the project directory and run `cd api` to CD into the `api` directory
- see the README.md file in the `api` directory for the preliminary steps
- ensure that a mysql database in installed and running on your device
- see the postman documentation on course enrollment and `Payments`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A
#### Questions:
